### PR TITLE
fix: wrong Glee version displayed in welcome message

### PR DESCRIPTION
**Pull request by fmvilas**

### CODE REVIEW

File: src/lib/logger.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+import { fileURLToPath } from 'url'
 import util from 'util'
 import chalk from 'chalk'
 import emojis from 'emojis'
 import wordWrap from 'word-wrap'
 
 export { chalk }
 
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = dirname(__filename)
```
Comment (magicmatatjahu) [inline src/lib/logger.js:12]: Why you use the `import.meta...` not classic `__dirname`?

Comment (fmvilas) [reply src/lib/logger.js:12]: https://nodejs.org/api/esm.html#esm_no_filename_or_dirname

Comment (fmvilas) [reply src/lib/logger.js:12]: This is one of the trade-offs of using ES Modules. I hope they come up with a solution soon :/

Comment (magicmatatjahu) [reply src/lib/logger.js:12]: LOL I always transform ESM to CJS in Node and I don't know about it. Good to know. Thanks!

**Review by magicmatatjahu — APPROVED** [review]

**Pull request merged**